main.cpp

```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+#include <cmath>
+
+bool checkRange(int _coef, int _lowRange, int _highRange) {
+	if (_coef < _lowRange || _coef > _highRange) {
+		std::cout << "You have entered incorrect number";
+		return false;
+	}
+	return true;
+}
+
+int main()
+{
+
+	constexpr int programmError{ -1 };
+
+	std::cout << "Please enter firts number: ";
+
+	constexpr int  kLowCoefRange{ 2 };
+	constexpr int  kHighCoefRange{ 255 };
+
+	int a{};
+	std::cin >> a;
+	if (checkRange(a, kLowCoefRange, kHighCoefRange) == false){
```

Comment:
проверку на bool редко кто явно делает на == true или == false 
чаще пишут просто 
```suggestion
	if (!checkRange(a, kLowCoefRange, kHighCoefRange)){
```
или 
```suggestion
	if ( not checkRange(a, kLowCoefRange, kHighCoefRange)){
```

Reply:
Agree